src/parsing/binast-deserializer.cc

```diff
@@ -182,34 +182,13 @@ BinAstDeserializer::DeserializeResult<AstConsString*> BinAstDeserializer::Deseri
   return {cons_string, offset};
 }
 
-void BinAstDeserializer::LinkUnresolvedVariableProxies() {
```

Comment:
Note that we no longer need to explicitly link unresolved VariableProxies together because we naturally can add them to the current scope in the DeserializeVariableProxy method. :tada: